scripts/tests/systemd/firezone-client.service

```diff
@@ -23,19 +23,19 @@ ProtectKernelModules=true
 ProtectKernelTunables=true
 # Docs say it's useless when running as root, but defense-in-depth
 ProtectProc=invisible
-ProtectSystem=full
+ProtectSystem=strict
 RestrictAddressFamilies=AF_INET AF_INET6 AF_NETLINK AF_UNIX
 RestrictNamespaces=true
 RestrictRealtime=true
 RestrictSUIDSGID=true
+StateDirectory=dev.firezone.client
```

Comment:
This asks systemd to make `/var/lib/dev.firezone.client` and mount it for us, since we're trying to minimize our own access to the FS